src/accelerate/commands/menu/selection_menu.py

```diff
@@ -15,13 +15,18 @@
 """
 Main driver for the selection menu, based on https://github.com/bchao1/bullet
 """
+import builtins
 import sys
 
+from ...utils.imports import _is_package_available
 from . import cursor, input
 from .helpers import Direction, clear_line, forceWrite, linebreak, move_cursor, reset_cursor, writeColor
 from .keymap import KEYMAP
 
 
+in_colab = _is_package_available("google.colab")
```

Comment:
Should be maybe be in a try except as it seems to fail instead of returning `False`.

Reply:
Yes, in some environment setup the code is not safe. When I test it in local console and colab, it works. 
I should wrap it with a try except.